core/src/main/scala/com/salesforce/op/OpWorkflowModelReader.scala

```diff
@@ -224,3 +230,30 @@ class OpWorkflowModelReader(val workflowOpt: Option[OpWorkflow]) extends MLReade
   }
 
 }
+
+private object FileReader {
+
+  def loadFile(pathString: String)(implicit conf: Configuration): String = {
```

Comment:
I think this should be moved to `com.salesforce.op.utils.io.hadoop` in `utils` sub-project.

Reply:
this is a not general util since it only works for things saved with 1 part file. I could modify it to work in general but it loads everything up into memory which is probably not something we want for most hadoop files. @gerashegalov any thoughts?

Reply:
The name could make that limitation clear, e.g. `loadSinglePartFile`.

Reply:
I will change the object name to indicate it is only fr workflow files

Reply:
I think we can wait with the cosmetics until there is a need to use it somewhere else unless we already see how to reuse this function. I want to first make sure we get the semantics right